bundles/org.openhab.binding.konnected/src/main/java/org/openhab/binding/konnected/internal/gson/KonnectedModuleGson.java

```diff
@@ -111,4 +122,26 @@ public String getAddr() {
     public void setAddr(String setAddr) {
         this.addr = setAddr;
     }
+
+    public void setZone(String ThingID, String zone) {
+        switch (ThingID) {
+            case PRO_MODULE:
+                this.setZone(zone);
+                break;
+            case WIFI_MODULE:
+                this.setPin(ESP8266_ZONE_TO_PIN.get(zone));
+                break;
+        }
+    }
+
+    public String getZone(String ThingID) {
+        switch (ThingID) {
+            case PRO_MODULE:
+                return this.getZone();
+
+            default:
+                return this.getPin();
+
```

Comment:
Remove linefeed.
```suggestion
```

Reply:
I don't like the inconsistency here, and this may be a bug. I think line 143 should be `return ESP8266_PIN_TO_ZONE.get(pin);`. And getPin() should just return the field.

Reply:
Fixed the zone mapping, and tested with a ESP8266 and a ESP32.